lib/pliny/commands/generator/schema.rb

```diff
@@ -38,34 +46,38 @@ def warn_legacy
         @warned_legacy = true
       end
 
+      def schema_path
+        if legacy?
+          "./docs/schema"
+        else
+          "./schema"
+        end
+      end
+
       def schema_json_path
         if legacy?
```

Comment:
Actually, can't we drop this conditional and rely on `schema_path` providing the right thing?